### ISSUE TITLE
Reserved funds not being released after identity clearance as stated in wiki

### ISSUE THREAD

**Issue by nymetva** [w3f/polkadot-wiki#982]
So the wiki under "Setting an identity" says "Users must reserve funds in a bond to store their information on chain - 10 KSM per identity, and 2.5 KSM per each field beyond the legal name. These funds are locked, not spent - they are returned when the identity is cleared."  

I had an identity cleared on West End and the 10 Westies reserved weren't unlocked at the completion of that process. I thought maybe I had to do something in an extrinsic but all I see that even looks close is clearing an identity to unlock funds.

In summary there seems to be a discrepansy between what the wiki says and the real life result when it comes to these reserved funds being unlocked after identity verification. 

Wiki here https://wiki.polkadot.network/docs/en/learn-identity

**Comment by Swader** [comment]
Please note that "cleared" means deleted, undone, and not approved by a registrar. The wiki is correct in saying that to get your funds back you need to clear your identity records.
